docs/readmes/howtos/ue_ip_address_management.md

```diff
@@ -95,6 +95,8 @@ first-come-first-served basis.
 
 ### DHCP
 
+NOTE: This feature requires installed `tcpdump`.
```

Comment:
do you mean libpcap dependency? I dont think DHCP allocation depends on `tcpdump` utility.

Reply:
Yep. You are right. It required `tcpdump` at the time of magma 1.2 and with the current version of `scapy` it requires just libpcap. 
Fixed